backend/tracim_backend/config.py

```diff
@@ -589,18 +589,45 @@ def _set_default_app(self, enabled_app_list: typing.List[str]):
         # TODO - G.M - 2018-08-08 - [GlobalVar] Refactor Global var
         # of tracim_backend, Be careful app_list is a global_var
         app_list.clear()
-        # calendar
-        if 'calendar' in enabled_app_list:
-            calendar = Application(
-                label='Calendar',
-                slug='calendar',
-                fa_icon='calendar',
-                is_active=False,
+
```

Comment:
@inkhey by the way of this MR, please refactor the code in order to separate concerns:

1. prepare a list of ready-to-use available applications
2. do the logic to make required applications available.

Eg.

```
available_apps = {
    'contents/html-document': Application(xxx),
    'contents/file': Application(yyy),
}

for name in enable_app_names:
    if name in available_apps.keys():
        enable_apps.append(available_apps[name]
```

This separate initialisation and processing

Reply:
Done with OrderedDict to preverse app ordering (goal of the PR)